Python/marshal.c

```diff
@@ -187,19 +187,20 @@ w_long(long x, WFILE *p)
         if ((n) > SIZE32_MAX) {                 \
             (p)->depth--;                       \
             (p)->error = WFERR_UNMARSHALLABLE;  \
-            return;                             \
+            return 0;                           \
```

Comment:
Some other functions in this file return -1 on error and 0 on success.

Reply:
`w_reserve` returns 0 on error and 1 on success, but it doesn't do any marshalling and don't use `W_SIZE`; `w_set`

Reply:
`w_ref` returns 0 on success and 1 on error; `w_reserved` does the opposite. But I think that if I will change return values of these functions as well, it will fit into scope of this PR.